__tests__/app.test.js

```diff
@@ -62,11 +66,137 @@ describe("GET/api/articles", () => {
+          .then(({ body: { article } }) => {
+            expect(article).toEqual({
+              article_id: 1,
+              title: "Living in the shadow of a great man",
+              topic: "mitch",
+              author: "butter_bridge",
+              body: "I find this existence challenging",
+              created_at: "2020-07-09T20:11:00.000Z",
+              votes: 110,
+            });
+          });
+      });
+      test("400: inc_votes is not a number", () => {
+        return request(app)
+          .patch("/api/articles/1")
+          .send({ inc_votes: "banana" })
+          .expect(400)
+          .then(({ body }) => {
+            expect(body.msg).toBe("Bad Request");
+          });
+      });
+    });
+    test("400: article id not in database", () => {
+      return request(app)
+        .get("/api/articles/1123")
+        .expect(400)
+        .then(({ body }) => {
+          expect(body.msg).toBe("Id not found");
+        });
+    });
+    test("400: wrong type of data", () => {
+      return request(app)
+        .get("/api/articles/banana")
+        .expect(400)
+        .then(({ body }) => {
+          expect(body.msg).toBe("Bad Request");
+        });
+    });
+  });
+});
+
+describe("/api/articles/:article_id/comments", () => {
+  describe("GET/api/articles/:article_id/comments", () => {
+    test("200: return comments with relevent article id", () => {
+      return request(app)
+        .get("/api/articles/1/comments")
+        .expect(200)
+        .then(({ body: comments }) => {
+          expect(comments).toHaveLength(11);
+          comments.forEach((comments) => {
+            expect(comments).toEqual(
+              expect.objectContaining({
+                comment_id: expect.any(Number),
+                author: expect.any(String),
+                body: expect.any(String),
+                created_at: expect.any(String),
+                votes: expect.any(Number),
+              })
+            );
+          });
+        });
+    });
+  });
+  describe(" POST/api/articles/:article_id/comments", () => {
+    test("201: create comment with relevent article id if username exists", () => {
+      return request(app)
+        .post("/api/articles/1/comments")
+        .send({ username: "rogersop", body: "Can he fix it?" })
+        .expect(201)
+        .then(({ body: comments }) => {
+          expect(comments).toEqual(
+            expect.objectContaining({
+              author: "rogersop",
+              body: "Can he fix it?",
+              article_id: 1,
+              comment_id: expect.any(Number),
+              created_at: expect.any(String),
+              votes: expect.any(Number)
+            })
+          )
+        });
+    });
+    test("400: throw error if username doesn't exist", () => {
+      return request(app)
+        .post("/api/articles/1/comments")
+        .send({ username: "bob the builder", body: "Can he fix it?" })
+        .expect(400)
+        .then(({ body: comments }) => {
```

Comment:
You're expecting one comment back from this post request so could you name it 'comment' for readability